### PR TITLE
[NUI] Fix the propertyNotification to be disposed in ScrollableBase

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Fix the propertyNotification to be disposed in ScrollableBase

### API Changes ###
none

### CODE REVIEW

**Review by taehyub — APPROVED** [review]
LGTM

**Review by Seoyeon2Kim — APPROVED** [review]
LGTM

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs [hunk 1]
```diff
@@ -1330,19 +1330,27 @@ protected override void Dispose(DisposeTypes type)
...
+            {
+                mPanGestureDetector.Detected -= OnPanGestureDetected;
+                mPanGestureDetector.Dispose();
+                mPanGestureDetector = null;
+            }
 
+            if (propertyNotification != null)
+            {
+                propertyNotification.Notified -= OnPropertyChanged;
+                propertyNotification.RemovePropertyNotifications();
```
Comment (hinohie) [inline src/Tizen.NUI.Components/Controls/ScrollableBase.cs:1346]: `RemovePropertyNotifications` is member API of `Animatable.cs`.  Did you mean this code?
```suggestion
                RemovePropertyNotifications();
```

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**